sdl-core/src/main/scala/io/smartdatalake/workflow/connection/authMode/SSLCertsAuthMode.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2024 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.workflow.connection.authMode
+
+import com.typesafe.config.Config
+import io.smartdatalake.config.{FromConfigFactory, InstanceRegistry}
+import io.smartdatalake.util.secrets.{SecretsUtil, StringOrSecret}
+
+/**
+ * Validate by SSL Certificates : Only location an credentials. Additional attributes should be
```

Comment:
What is meant by "Only location an credentials" ?

Reply:
This is not a new comment, just refactoring in a seperate class ;-)
I will improve it...